go/vt/sqlparser/sql.y

```diff
@@ -5941,6 +6107,38 @@ ins_column_list:
   {
     $$ = append($$, $5)
   }
+| non_reserved_keyword2
+  {
+    $$ = Columns{NewColIdent(string($1))}
+  }
+| ACCOUNT
```

Comment:
Would be better to put all these keywords in a rule like the above and match that, rather than name every one here. Did you try that?